packages/design-system/src/styles/settings/variables/_reference.scss

```diff
@@ -0,0 +1,15 @@
+// these variables are reference names for backwards compatibility with
```

Comment:
These files are not linked in the index.scss file for this pr, but will be when themes are connected.